### PR TITLE
Fix cling-bytecode-runhenry on Windows 64

### PR DESCRIPTION
**Pull request by bellenot**
Fix an error when using fscanf on Windows 64 bit:
```
1141: Processing C:/Users/sftnight/git/roottest/cling/bytecode/runhenry.C...
1141:
1141: -- END TEST OUTPUT --
1141: CMake Error at C:/Users/sftnight/build/release/RootTestDriver.cmake:181 (message):
1141:   got exit code Exit code 0xc0000409
1141:
1141:    but expected 0
1141:
1141:
1/1 Test #1141: roottest-cling-bytecode-runhenry ...***Failed    2.13 sec
```

### CODE REVIEW

**Comment by bellenot** [comment]
@phsft-bot build just on windows10/cxx14

**Comment by bellenot** [comment]
@phsft-bot build just on windows10/cxx14

File: cling/bytecode/runhenry.C [hunk 1]
```diff
@@ -32,11 +32,13 @@ void runhenry(){
...
-  FILE *data;
   char datafname[]="henry.dat";
   
-  data = fopen(datafname,"r");
-  while (fscanf(data, "%lf %lf %lf %lf",&x,&xE,&y,&yE) != EOF) {
+  std::ifstream in(datafname);
+  while (in.is_open() && !in.eof()) {
+    in >> x >> xE >> y >> yE;
+    if (!in.good())
+      break;
```
Comment (Axel-Naumann) [inline cling/bytecode/runhenry.C:41]: Isn't this enough?
```suggestion
  while (in >> x >> xE >> y >> yE) {
```

**Review by Axel-Naumann — APPROVED** [review]
See comment!

File: cling/bytecode/runhenry.C (hunk 1 above)
Comment (bellenot) [reply cling/bytecode/runhenry.C:41]: Yes, it is, thanks! :-)

**Comment by bellenot** [comment]
OK, the underlying issue has been found and the PR https://github.com/root-project/root/pull/9973 is on its way

**Pull request closed without merging**